src/templates/mkt_nav.html

```diff
@@ -15,6 +24,18 @@
     {{ nav_item(url('new'), _('New'), 'new') }}
     {{ nav_item(url('recommended'), _('Recommended'), 'recommended') }}
     {{ nav_item('#', _('Categories'), 'categories', 'categories') }}
+    <div class="mkt-nav-root--divider"></div>
```

Comment:
Yeah, later on we might want to separate them completely while finding out ways not to completely repeat the links within both navs (maybe a JS object to populate them both? wrapper element?).